src/collections/_documentation/platforms/react-native/touchevents.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Tracking Touch Events'
+---
+
+{% version_added 1.5.0 %}
+
+To track touch events with the React Native SDK you will need at least version `1.5.0` and will have to wrap your app with a `TouchEventBoundary`. This can be done in two ways:
```

Comment:
What happens if multiple `TouchEventBoundaries` are used? Should we put a warning against that?

Reply:
You can actually have multiple if you want to wrap specific parts of the app, but they should not be nested. Good catch, will add that.